internal/pkg/skuba/upgrade/node/versions.go

```diff
@@ -99,6 +102,10 @@ func (nviu NodeVersionInfoUpdate) NodeUpgradeableCheck(client clientset.Interfac
 	if err != nil {
 		return err
 	}
+	//Check if node OS SP compatible with kubernetes.latestVersion
+	if strings.Contains(nviu.Current.Node.Status.NodeInfo.OSImage, "SUSE Linux Enterprise Server 15 SP1") {
+		log.Fatalf("You are still running 15 SP1 on your node. Please upgrade to kubernetes version %s on SP1 first, and migrate to SP2 before triggering this upgrade.", skuba.LastCaaSP4KubernetesVersion)
```

Comment:
do not log.Fatalf, instead, append the error into the list of error messages.

Reply:
Changed.